framework/flamingo/sessions.go

```diff
@@ -224,6 +233,7 @@ flamingo: session: {
 	redis: {
 		url: string | *""
 		host: string | *"redis"
+		username?: string & !=""
```

Comment:
do we want to go away from the usual `username: string | *""`?
After injection, it will be an empty string anyway